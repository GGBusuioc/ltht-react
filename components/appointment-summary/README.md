
# AppointmentSummary

<!-- STORY -->

### Import

```js
import AppointmentSummary from '@ltht-react/appointment-summary'
```

### Usage

```jsx
<AppointmentSummary />
```
  