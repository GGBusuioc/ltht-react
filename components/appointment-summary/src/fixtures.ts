import { Encounter, EncounterStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const AppointmentOne: Encounter = {
  id: '346ca927-67e0-e931-a2c7-115087226fg2|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'See GP regarding sore throat',
      coding: [{ display: 'GP', code: '64706' }],
    },
  ],
  period: {
    start: {
      value: '2020-04-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Beeston Village Surgery',
    typeName: 'GP',
  },
  status: EncounterStatusCode.Planned,
}

const AppointmentTwo: Encounter = {
  id: '941ca927-34e0-e911-a2c7-005087226fe4|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'Chest x-ray for recent car accident',
      coding: [{ display: 'Hospital', code: '26706' }],
    },
  ],
  period: {
    start: {
      value: '2019-12-12T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Chapel Allerton Hospital',
    typeName: 'Hospital',
  },
  status: EncounterStatusCode.Finished,
}

export { AppointmentOne, AppointmentTwo }
