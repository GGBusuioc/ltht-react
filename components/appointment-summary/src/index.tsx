import React from 'react'

import { Encounter } from '@ltht-react/types'
import { Widget, WidgetHeader, WidgetList, WidgetListItem } from '@ltht-react/widget'
import AppointmentSummaryItem from './molecules/appointment-summary-item'

const AppointmentSummary: React.FC<Props> = ({ title = 'Appointments', encounters = [], clickHandler }) => {
  return (
    <Widget noData={encounters.length === 0}>
      <WidgetHeader>
        <h3>{title}</h3>
      </WidgetHeader>
      <WidgetList clickable={!!clickHandler}>
        {encounters.map(encounter => (
          <WidgetListItem key={encounter.id}>
            <AppointmentSummaryItem encounter={encounter} clickHandler={clickHandler} />
          </WidgetListItem>
        ))}
      </WidgetList>
    </Widget>
  )
}

interface Props {
  title?: string
  encounters: Encounter[] | undefined
  clickHandler?(appointment: Encounter): void
}

export default AppointmentSummary
