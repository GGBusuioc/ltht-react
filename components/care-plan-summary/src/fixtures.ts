import { CarePlan, CarePlanStatusCode, PartialDateTimeKindCode, CarePlanIntentCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CarePlanOne: CarePlan = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|careplan',
  metadata: mockMetadata,
  author: { display: 'Joe Bloggs', type: 'Test' },
  intent: CarePlanIntentCode.Plan,
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Active,
  title: 'Test Care Plan Summary One',
}

const CarePlanTwo: CarePlan = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|careplan',
  metadata: mockMetadata,
  author: { display: 'John Mclane', type: 'Test' },
  intent: CarePlanIntentCode.Proposal,
  period: {
    start: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2015-03-04T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Completed,
  title: 'Test Care Plan Summary Two',
}

export { CarePlanOne, CarePlanTwo }
