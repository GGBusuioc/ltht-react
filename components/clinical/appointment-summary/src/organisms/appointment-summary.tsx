import React from 'react'
import styled from '@emotion/styled'

import { Encounter } from '@ltht-react/types'
import { DateSummary } from '@ltht-react/summary'

import ServiceProvider from '../atoms/appointment-service-provider'
import Description from '../atoms/appointment-description'
import Status from '../atoms/appointment-status'
import Redacted from '../molecules/appointment-redacted'

const StyledSummary = styled.div`
  display: flex;
  justify-content: center;
`
const StyledDescription = styled.div`
  flex: 1;
`
const StyledDate = styled.div`
  flex: 1;
  text-align: left;
`
const StyledService = styled.div`
  flex: 1;
  text-align: right;
`

const AppointmentSummary: React.FC<Props> = ({ encounter }) => {
  if (encounter.metadata.isRedacted) {
    return (
      <StyledSummary>
        <Redacted appointment={encounter} />
      </StyledSummary>
    )
  }

  return (
    <StyledSummary>
      <StyledDate>
        <DateSummary datetime={encounter?.period?.start} />
      </StyledDate>
      <StyledDescription>
        <Description encounter={encounter} />
      </StyledDescription>
      <StyledService>
        <ServiceProvider encounter={encounter} />
        <Status encounter={encounter} />
      </StyledService>
    </StyledSummary>
  )
}

interface Props {
  encounter: Encounter
}

export default AppointmentSummary
