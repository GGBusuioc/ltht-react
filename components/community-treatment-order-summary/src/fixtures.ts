import {
  LypftCommunityTreatmentOrder,
  CarePlanStatusCode,
  PartialDateTimeKindCode,
  CarePlanIntentCode,
  Metadata,
} from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CommunityTreatmentOrderOne: LypftCommunityTreatmentOrder = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Not Given',
  legalStatus: {
    coding: [{ display: 'Approved', code: '3309007' }],
    text: 'Approved',
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No restrictions',
}

const CommunityTreatmentOrderTwo: LypftCommunityTreatmentOrder = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Given',
  legalStatus: {
    coding: [{ display: 'Pending', code: '779007' }],
    text: 'Pending',
  },
  period: {
    start: {
      value: '2020-01-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No Restrictions',
}

export { CommunityTreatmentOrderOne, CommunityTreatmentOrderTwo }
