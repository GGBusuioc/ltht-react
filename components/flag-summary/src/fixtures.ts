import { Flag, FlagStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const FlagOne: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'Lives in Known High Risk Area', code: '109006' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Year,
    },
  },
  status: FlagStatusCode.Inactive,
}

const FlagTwo: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'High Risk of Violence', code: '56706' }],
  },
  category: {
    coding: [{ display: 'Violence', code: '66706' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },

  status: FlagStatusCode.Inactive,
}

export { FlagOne, FlagTwo }
