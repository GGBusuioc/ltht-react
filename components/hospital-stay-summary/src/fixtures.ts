import { HospitalStay, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const HospitalStayOne: HospitalStay = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '123456789',
  serviceProvider: { display: 'Chapel Allerton Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

const HospitalStayTwo: HospitalStay = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '987654321',
  serviceProvider: { display: 'Beaumont Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2020-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

export { HospitalStayOne, HospitalStayTwo }
