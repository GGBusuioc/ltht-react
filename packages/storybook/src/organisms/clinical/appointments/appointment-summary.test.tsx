import React from 'react'
import { mount } from 'enzyme'
import ReactDOM from 'react-dom'

import AppointmentSummary from '@ltht-react/appointment-summary'
import { AppointmentOne, AppointmentTwo } from './appointment-summary.fixtures'

describe('AppointmentSummary', () => {
  const appointments = [AppointmentOne, AppointmentTwo]
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<AppointmentSummary title="Appointments" encounters={appointments} />, div)
  })
  it('matches snapshot', () => {
    expect(mount(<AppointmentSummary title="Appointments" encounters={appointments} />)).toMatchSnapshot(
      'wrapper mount'
    )
  })
})
