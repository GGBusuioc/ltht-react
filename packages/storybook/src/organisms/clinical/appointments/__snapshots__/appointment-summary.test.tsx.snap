// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppointmentSummary matches snapshot: wrapper mount 1`] = `
.emotion-6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.emotion-1 {
  text-align: left;
}

.emotion-0 {
  color: #231f20;
}

.emotion-3 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.emotion-2 {
  color: #231f20;
  text-align: left;
  margin-left: 0.5em;
}

.emotion-4 {
  color: -v-a-l-u-e:#768692;
  -d-a-r-k-e-r25: #59656e;
  -l-i-g-h-t-e-r25: #98a4ad;
  text-align: right;
}

.emotion-5 {
  color: -v-a-l-u-e:#768692;
  -d-a-r-k-e-r25: #59656e;
  -l-i-g-h-t-e-r25: #98a4ad;
  text-align: right;
}

<AppointmentSummary
  encounters={
    Array [
      Object {
        "id": "346ca927-67e0-e931-a2c7-115087226fg2|appointment",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
          "isRedacted": true,
          "requestedWhen": "",
        },
        "period": Object {
          "start": Object {
            "kind": "DATE",
            "value": "2020-04-01T00:00:00+00:00",
          },
        },
        "serviceProvider": Object {
          "display": "Beeston Village Surgery",
          "typeName": "GP",
        },
        "status": "PLANNED",
        "text": Object {
          "div": "test",
          "text": "See GP regarding sore throat",
        },
        "type": Array [
          Object {
            "coding": Array [
              Object {
                "code": "64706",
                "display": "GP",
              },
            ],
            "text": "See GP regarding sore throat",
          },
        ],
      },
      Object {
        "id": "941ca927-34e0-e911-a2c7-005087226fe4|appointment",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
          "isRedacted": true,
          "requestedWhen": "",
        },
        "period": Object {
          "start": Object {
            "kind": "DATE",
            "value": "2019-12-12T00:00:00+00:00",
          },
        },
        "serviceProvider": Object {
          "display": "Chapel Allerton Hospital",
          "typeName": "Hospital",
        },
        "status": "FINISHED",
        "text": Object {
          "div": "test",
          "text": "Chest x-ray for recent car accident",
        },
        "type": Array [
          Object {
            "coding": Array [
              Object {
                "code": "26706",
                "display": "Hospital",
              },
            ],
            "text": "Chest x-ray for recent car accident",
          },
        ],
      },
    ]
  }
>
  <ListItem
    key="346ca927-67e0-e931-a2c7-115087226fg2|appointment"
  >
    <li>
      <AppointmentSummaryItem
        encounter={
          Object {
            "id": "346ca927-67e0-e931-a2c7-115087226fg2|appointment",
            "metadata": Object {
              "dataSources": Array [
                Object {
                  "display": "Mock",
                },
              ],
              "isRedacted": true,
              "requestedWhen": "",
            },
            "period": Object {
              "start": Object {
                "kind": "DATE",
                "value": "2020-04-01T00:00:00+00:00",
              },
            },
            "serviceProvider": Object {
              "display": "Beeston Village Surgery",
              "typeName": "GP",
            },
            "status": "PLANNED",
            "text": Object {
              "div": "test",
              "text": "See GP regarding sore throat",
            },
            "type": Array [
              Object {
                "coding": Array [
                  Object {
                    "code": "64706",
                    "display": "GP",
                  },
                ],
                "text": "See GP regarding sore throat",
              },
            ],
          }
        }
      >
        <div
          className="emotion-6"
          role="link"
        >
          <div
            className="emotion-1"
          >
            <DateSummary
              datetime={
                Object {
                  "kind": "DATE",
                  "value": "2020-04-01T00:00:00+00:00",
                }
              }
            >
              <div
                className="emotion-0"
              >
                01-Apr-2020
              </div>
            </DateSummary>
          </div>
          <div
            className="emotion-3"
          >
            <AppointmentDescription
              encounter={
                Object {
                  "id": "346ca927-67e0-e931-a2c7-115087226fg2|appointment",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "start": Object {
                      "kind": "DATE",
                      "value": "2020-04-01T00:00:00+00:00",
                    },
                  },
                  "serviceProvider": Object {
                    "display": "Beeston Village Surgery",
                    "typeName": "GP",
                  },
                  "status": "PLANNED",
                  "text": Object {
                    "div": "test",
                    "text": "See GP regarding sore throat",
                  },
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "64706",
                          "display": "GP",
                        },
                      ],
                      "text": "See GP regarding sore throat",
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-2"
              >
                See GP regarding sore throat
              </div>
            </AppointmentDescription>
          </div>
          <div>
            <AppointmentServiceProvider
              encounter={
                Object {
                  "id": "346ca927-67e0-e931-a2c7-115087226fg2|appointment",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "start": Object {
                      "kind": "DATE",
                      "value": "2020-04-01T00:00:00+00:00",
                    },
                  },
                  "serviceProvider": Object {
                    "display": "Beeston Village Surgery",
                    "typeName": "GP",
                  },
                  "status": "PLANNED",
                  "text": Object {
                    "div": "test",
                    "text": "See GP regarding sore throat",
                  },
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "64706",
                          "display": "GP",
                        },
                      ],
                      "text": "See GP regarding sore throat",
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-4"
              >
                Beeston Village Surgery
              </div>
            </AppointmentServiceProvider>
            <AppointmentStatus
              encounter={
                Object {
                  "id": "346ca927-67e0-e931-a2c7-115087226fg2|appointment",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "start": Object {
                      "kind": "DATE",
                      "value": "2020-04-01T00:00:00+00:00",
                    },
                  },
                  "serviceProvider": Object {
                    "display": "Beeston Village Surgery",
                    "typeName": "GP",
                  },
                  "status": "PLANNED",
                  "text": Object {
                    "div": "test",
                    "text": "See GP regarding sore throat",
                  },
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "64706",
                          "display": "GP",
                        },
                      ],
                      "text": "See GP regarding sore throat",
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-5"
              >
                Planned
              </div>
            </AppointmentStatus>
          </div>
        </div>
      </AppointmentSummaryItem>
    </li>
  </ListItem>
  <ListItem
    key="941ca927-34e0-e911-a2c7-005087226fe4|appointment"
  >
    <li>
      <AppointmentSummaryItem
        encounter={
          Object {
            "id": "941ca927-34e0-e911-a2c7-005087226fe4|appointment",
            "metadata": Object {
              "dataSources": Array [
                Object {
                  "display": "Mock",
                },
              ],
              "isRedacted": true,
              "requestedWhen": "",
            },
            "period": Object {
              "start": Object {
                "kind": "DATE",
                "value": "2019-12-12T00:00:00+00:00",
              },
            },
            "serviceProvider": Object {
              "display": "Chapel Allerton Hospital",
              "typeName": "Hospital",
            },
            "status": "FINISHED",
            "text": Object {
              "div": "test",
              "text": "Chest x-ray for recent car accident",
            },
            "type": Array [
              Object {
                "coding": Array [
                  Object {
                    "code": "26706",
                    "display": "Hospital",
                  },
                ],
                "text": "Chest x-ray for recent car accident",
              },
            ],
          }
        }
      >
        <div
          className="emotion-6"
          role="link"
        >
          <div
            className="emotion-1"
          >
            <DateSummary
              datetime={
                Object {
                  "kind": "DATE",
                  "value": "2019-12-12T00:00:00+00:00",
                }
              }
            >
              <div
                className="emotion-0"
              >
                12-Dec-2019
              </div>
            </DateSummary>
          </div>
          <div
            className="emotion-3"
          >
            <AppointmentDescription
              encounter={
                Object {
                  "id": "941ca927-34e0-e911-a2c7-005087226fe4|appointment",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "start": Object {
                      "kind": "DATE",
                      "value": "2019-12-12T00:00:00+00:00",
                    },
                  },
                  "serviceProvider": Object {
                    "display": "Chapel Allerton Hospital",
                    "typeName": "Hospital",
                  },
                  "status": "FINISHED",
                  "text": Object {
                    "div": "test",
                    "text": "Chest x-ray for recent car accident",
                  },
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "26706",
                          "display": "Hospital",
                        },
                      ],
                      "text": "Chest x-ray for recent car accident",
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-2"
              >
                Chest x-ray for recent car accident
              </div>
            </AppointmentDescription>
          </div>
          <div>
            <AppointmentServiceProvider
              encounter={
                Object {
                  "id": "941ca927-34e0-e911-a2c7-005087226fe4|appointment",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "start": Object {
                      "kind": "DATE",
                      "value": "2019-12-12T00:00:00+00:00",
                    },
                  },
                  "serviceProvider": Object {
                    "display": "Chapel Allerton Hospital",
                    "typeName": "Hospital",
                  },
                  "status": "FINISHED",
                  "text": Object {
                    "div": "test",
                    "text": "Chest x-ray for recent car accident",
                  },
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "26706",
                          "display": "Hospital",
                        },
                      ],
                      "text": "Chest x-ray for recent car accident",
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-4"
              >
                Chapel Allerton Hospital
              </div>
            </AppointmentServiceProvider>
            <AppointmentStatus
              encounter={
                Object {
                  "id": "941ca927-34e0-e911-a2c7-005087226fe4|appointment",
                  "metadata": Object {
                    "dataSources": Array [
                      Object {
                        "display": "Mock",
                      },
                    ],
                    "isRedacted": true,
                    "requestedWhen": "",
                  },
                  "period": Object {
                    "start": Object {
                      "kind": "DATE",
                      "value": "2019-12-12T00:00:00+00:00",
                    },
                  },
                  "serviceProvider": Object {
                    "display": "Chapel Allerton Hospital",
                    "typeName": "Hospital",
                  },
                  "status": "FINISHED",
                  "text": Object {
                    "div": "test",
                    "text": "Chest x-ray for recent car accident",
                  },
                  "type": Array [
                    Object {
                      "coding": Array [
                        Object {
                          "code": "26706",
                          "display": "Hospital",
                        },
                      ],
                      "text": "Chest x-ray for recent car accident",
                    },
                  ],
                }
              }
            >
              <div
                className="emotion-5"
              >
                Finished
              </div>
            </AppointmentStatus>
          </div>
        </div>
      </AppointmentSummaryItem>
    </li>
  </ListItem>
</AppointmentSummary>
`;
